pandas/tests/io/excel/test_readers.py

```diff
@@ -969,6 +998,14 @@ def test_reader_seconds(self, request, engine, read_ext):
                 )
             )
 
+        # GH 55045
+        if engine == "calamine" and read_ext == ".ods":
+            request.node.add_marker(
+                pytest.mark.xfail(
+                    reason="ODS file contains bad datetime (seconds as text)"
+                )
+            )
```

Comment:
Similar - test the exception if calamine is raising one.

Reply:
Calamine doesn't raise exceptions, I described problems with this test in https://github.com/pandas-dev/pandas/issues/55045. ODFReader extracts time from cell text, not XML attribute `office:time-value`, as written in the specification and as LibreOffice/Google Sheets/calamine do.